.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: ruby
 sudo: false
+before_install:
+  - gem update --system 2.7.7
```

Comment:
I haven’t seen this before. Is this installing a specific version of rubygems? If so I see why so it’s consistent for all builds. Normally though it’s the most recent. Are we sure we want this? I take it we’d have to do this in other repos too?

Reply:
@deivid-rodriguez perhaps its not a big deal, but I noticed the `before_install` section here does not install the latest bundler [like we do in activeadmin](https://github.com/activeadmin/activeadmin/blob/96b22bf08a879dc856aee0f7396d2a70fbc65209/.travis.yml#L29). Do you think we should have that here and be consistent or just this as is? Let me know what you think as besides that I think this looks good to go. 👍🏻